modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -21,6 +21,7 @@ object ObservationDao {
                     ${o.id.index},
                     ${o.title},
                     ${o.instrument})
+       ON CONFLICT DO NOTHING
```

Comment:
At the moment, I don't have any program or observation information other than the id.  Maybe what we want is to do a query and check whether the program and observation already exist before trying to create stubbed out versions.